moban/jinja2/engine.py

```diff
@@ -100,6 +100,9 @@ def get_template(self, template_file):
         template file exists at:
             '/User/moban-pro/my-template/templates/myfile.jj2'
         """
+        if custom_env_vars:
```

Comment:
custom env vars could be passed via jinja2's global plugin.

Reply:
https://github.com/moremoban/moban/blob/dev/moban/jinja2/extensions.py#L31

Reply:
I think a concern is that each target could have different options.

tbis needs to nave tests written *before* it is reviewable. Tests communicate the goalsof a PR.